src/event/cmap.rs

```diff
@@ -230,41 +230,41 @@ pub struct ConnectionCheckedInEvent {
 pub trait CmapEventHandler: Send + Sync {
     /// A [`Client`](../../struct.Client.html) will call this method on each registered handler
     /// whenever a connection pool is created.
-    fn handle_pool_created_event(&self, event: PoolCreatedEvent) {}
+    fn handle_pool_created_event(&self, _event: PoolCreatedEvent) {}
```

Comment:
Question: Why not just use _ instead of _event?

Reply:
These are methods on a public trait, so I think it's useful to have names for the parameters to ease readability for users looking at the docs.